src/olympia/amo/tests/__init__.py

```diff
@@ -678,6 +682,14 @@ def addon_factory(
     for user in users:
         addon.addonuser_set.create(user=user)
 
+    application = version_kw.get('application', amo.FIREFOX.id)
+    if not category:
+        category = Category.from_static_category(
+            next(CATEGORIES[application][type_].itervalues()))
```

Comment:
Also, how about just picking a category at random ? Not sure if it's a good idea...